eox_hooks/actions.py

```diff
@@ -112,3 +113,40 @@ def get_request_fields(fields, extra_fields, **kwargs):
         data[name] = field
 
     return data
+
+
+def trigger_enrollments_creation(**kwargs):
+    """
+    Custom action that starts an async task that enrolls a user in a list of courses.
+
+    After a user's enrollment to a Course Program, this action ensures that the
+    user is also enrolled in the courses that define the program.
+
+    If the course does not have the setting EDNX_TRIGGER_FOLLOWUP_ENROLLMENTS
+    in its other_course_settings, then no other enrollment is created.
+
+    The setting should look like this:
+
+    "EDNX_TRIGGER_FOLLOWUP_ENROLLMENTS": [
+        {
+            "course_id": "course-v1:Demo+CSTest+2020",
+            "mode": "honor",
+            "force": true
```

Comment:
There's no force option isn't it? or did I miss it somewhere?

Reply:
This was on my to-do list to remove. Thanks